src/main.ts

```diff
@@ -24,17 +25,22 @@ async function run(): Promise<void> {
 
     if (!issueKey) {
       console.log(`JIRA key was not found`);
-      process.exit(0);
+      if (FAIL_PR_WHEN_JIRA_ISSUE_NOT_FOUND === 'true') {
```

Comment:
it should be casted to bool in `getInputs` method

Reply:
Done